packages/starlight/utils/route-data.ts

```diff
@@ -85,7 +83,7 @@ function getLastUpdated({ entry }: PageProps): Date | undefined {
 		try {
 			return frontmatterLastUpdated instanceof Date
 				? frontmatterLastUpdated
-				: getNewestCommitDate(entry.id);
+				: getNewestCommitDate(entry.filePath);
```

Comment:
I guess for now we’re still only supporting on-disk `src/content/` documents, so no need to handle entries which don’t include `filePath`? (i.e. remote content loaders)

Reply:
Yeah, the typechecking will fail when we make `filePath` optional in our `StarlightDocsEntry` type and we would need to take care of that when doing so.

Reply:
In general I guess this will also fall into the category of “stuff we need to know about loaders”, so we can toggle features (e.g. Git dates) based on that.